helm/botkube/e2e-test-values.yaml

```diff
@@ -20,7 +20,7 @@ communications:
             executors:
```

Comment:
Please update the `values.yaml` file for Helm chart to indicate the webhook also supports bindings:

```yaml
    ## Settings for Webhook.
    webhook:
      # -- If true, enables Webhook.
      enabled: false
      # -- The Webhook URL, e.g.: https://example.com:80
      url: 'WEBHOOK_URL'
      bindings:
        # -- Notification sources configuration for the webhook.
        sources:
          - k8s-events
```

Then, regenerate the readme for Helm chart. Thanks!

Reply:
Make sense.

Reply:
Done.